backend/app/scripts/migrate_passwords_to_bcrypt.py

```
@@ -0,0 +1,290 @@
+
+    def run_full_migration(self) -> dict[str, Any]:
+        """运行完整迁移"""
+        logger.info(f"开始密码迁移 - {'DRY RUN' if self.dry_run else 'EXECUTE'} 模式")
+        logger.info(f"批次大小: {self.batch_size}")
+
+        self.stats["start_time"] = datetime.now()
+
+        with Session(engine) as session:
+            # 获取总用户数
+            total_statement = select(User).where(
+                User.is_active == True,
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)